### ISSUE TITLE
During the fight with the quest jailer ship, crew rescued via the players teleporter are re-rescused, no matter where they are. 

### ISSUE THREAD

**Issue by Duodecimus** [arcburnergit/RAD-FTL-MV-Addon#12]
When the jailer beams hit a crew it abducts them to the enemy ship.
A while later, the crew is returned onto your ship.

If the player uses their own teleporter to retrieve the crew this timer is not reset.
In addition, having been returned to the players ship the crew is now eligible to be hit by the beam again. This starts a new timer, without removing the old one.

The combination of these two mechanics can end up with player crew getting multiple stacks of the return timer, causing them to start bouncing between captured and return to the ship, and returned to the ship from the ship.

Basically what I'm saying is that player teleporting should remove the return timer, and you should not apply a timer to a crew that already has one

**Comment by arcburnergit** [comment]
Fixed.
